ops/ansible/playbooks-ccs/launch_bfd-server-load.yml

```diff
@@ -0,0 +1,41 @@
+- name: Configure server-load Controller
+  hosts: localhost
+
+  tasks:
+    - name: Apply server-load Role
+      import_role:
+        name: bfd-server-load
+      vars:
+        # Variables listed are applied from user-data.sh inside the server-load terraform module
+        #server_load_sqs_queue_name:
+        #server_load_node_lambda_name:
+        #server_load_asg_name:
+        #server_load_test_host:
+        #server_load_aws_current_region:
+        #server_load_initial_worker_nodes:
+        #server_load_node_spawn_time:
+        #server_load_max_spawned_nodes:
+        #server_load_max_spawned_users:
+        #server_load_user_spawn_rate:
+        #server_load_test_runtime_limit:
+        #server_load_coasting_time:
+        #server_load_warm_instance_target:
+        #server_load_stop_on_scaling:
+        #server_load_stop_on_node_limit:
```

Comment:
The comment on variable injection via user-data is probably fine, but you might consider dropping the other commented-out lines of variables.

Reply:
As usual, I don't feel strongly either way -- this was more-or-less me "cargo culting" the existing Ansible playbooks I built upon. It was already a bit difficult to keep track of everywhere these variables were referenced, so this list is very likely to end up out-of-sync.

I'll remove this list but keep the comment indicating where the variables for the task are set.